pcsx2/VMManager.cpp

```diff
@@ -713,6 +728,11 @@ bool VMManager::Initialize(const VMBootParameters& boot_params)
 	frameLimitReset();
 	cpuReset();
 
+#ifdef ENABLE_ACHIEVEMENTS
+	if (EmuConfig.Achievements.Enabled)
```

Comment:
needs to also check `!Achievements::IsUsingRAIntegration()` to prevent assertion in `Achievements::Initialize`

Reply:
Moved the check inside `Achievements`.